lava-job-runner.py

```diff
@@ -1,4 +1,22 @@
 #!/usr/bin/python
+#
+# Copyright (C) 2014, 2015, 2016 Linaro Limited
```

Comment:
Sure, so I'll push an updated change with all the authors, then ask the founders to confirm they're happy with the license (should be fine as it has been previously discussed).